### PR TITLE
add profileImageAnimation

### PR DESCRIPTION
**Pull request by thiunuwan**
Simple animation for profile images.

### CODE REVIEW

File: Art/thiunuwan/styles.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  background-color: #121212;
+}
+
+.hero__image {
+  outline: 4px solid #1ed9b7;
+  /* background-image: url('img/me.png'); */
+  display: block;
+  width: 350px;
+  height: auto;
+  /*box-shadow: inset 0 0 0 9px rgb(203, 37, 37);*/
```
Comment (LaurelineP) [inline Art/thiunuwan/styles.css:16]: # Feedbacks
- [ ] Unnecessary comments
Please remove the unnecessaries comments on l:12 and l:16
# Context
```css
/* background-image: url('img/me.png'); 


/*box-shadow: inset 0 0 0 9px rgb(203, 37, 37);*/

```

**Review by LaurelineP — CHANGES_REQUESTED** [review]

**Comment by LaurelineP** [comment]
Thanks @thiunuwan for your contributions and nice work ! 🙌

I've added some review to address before merging your PR 
Please address them in order to proceed to the merge of your branch :)

**Review by thiunuwan — COMMENTED** [review]
changes done

**Review by LaurelineP — APPROVED** [review]
@thiunuwan Thanks for your reactivity ! 🙌
And this is all good for me 👍

**Pull request merged**